sympy/matrices/matrices.py

```diff
@@ -340,9 +340,9 @@ def inv(self, method=None, **kwargs):
         Inverse is found using Gaussian elimination & LU, with former being default.
 
         Raises
-        ========
-
-        If the determinant of the matrix is zero, a ``ValueError`` is raised.
+        ------
+        ValueError
+        If the determinant of the matrix is zero.
```

Comment:
It should somehow be indicated that the methods are different for sparse and dense matrices. Also their defaults are different. Otherwise, it looks good.

So I think that no default should be mentioned in the short summary on the first line, and the the methods should be separated into two groups according to the type of the matrix.